taar/recommenders/__init__.py

```diff
@@ -1,11 +1,14 @@
 from .collaborative_recommender import CollaborativeRecommender
 from .empty_recommender import EmptyRecommender
 from .locale_recommender import LocaleRecommender
+from .legacy_recommender import LegacyRecommender
 from .recommendation_manager import RecommendationManager
 
+
 __all__ = [
     'CollaborativeRecommender',
     'EmptyRecommender',
     'LocaleRecommender',
+    'LegacyRecommender',
```

Comment:
nit: let's move this above `LocaleRecommender` to respect the alphabetical ordering of the array